lib/auth/new_web_user.go

```diff
@@ -263,6 +279,14 @@ func (s *AuthServer) CreateUserWithOTP(token string, password string, otpToken s
 
 // CreateUserWithoutOTP creates an account with the provided password and deletes the token afterwards.
 func (s *AuthServer) CreateUserWithoutOTP(token string, password string) (services.WebSession, error) {
+	clusterConfig, err := s.GetClusterConfig()
```

Comment:
From the first glance it looks like it should be handled using AuthPreferences rather than a stand alone configuration property.

Reply:
I think we are moving auth preferences in the cluster config in the longer term, so kind of makes sense